packages/mosaic/tooltip/tooltip.scss

```diff
@@ -6,9 +6,9 @@
 
 $zindex-tooltip: 1060;
 $tooltip-max-width: 240px;
-$border-radius-base: 3px;
-$tooltip-arrow-width: 8px;
-$tooltip-distance: $tooltip-arrow-width + 3px;
+$border-radius-base: 4px;
```

Comment:
why did you change it ?
on sketch border-radius: 3px;
https://app.zeplin.io/project/5b902f859dd1892babf00694/screen/5afb0c9f9a9ea13f3664e95f

Reply:
Right, it's my mistake.